src/main/java/org/junit/runners/MethodSorters.java

```diff
@@ -25,6 +26,13 @@
     JVM(null),
 
     /**
+     * Sorts the test methods in pseudo-random order.
+     * This sort option generates and displays (to stdout) a
+     * seed value for the random number generator.
+     */
+    RANDOM(null),
```

Comment:
I just pushed up a commit that fixes this (just had to change line 33 to use the DEFAULT ordering--since in MethodSorter, the existing line to do the sort is before the shuffle takes place). Thanks for the comments, by the way.